docs/docs/build/dashboards/dashboards-101.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Dashboards in Rill
+description: Understanding Explore vs Canvas dashboards and when to use each
+sidebar_label: Dashboard Types
+sidebar_position: 00
+---
+
+Rill offers two distinct dashboard experiences, each optimized for different use cases and user needs. The key difference lies in how they handle **metrics views** - the data sources that power your dashboards.
+
+- **Explore Dashboards** - slice-and-dice exploratory dashboard to find insights into your data```
+- **Canvas Dashboards** - traditional mutli-component graph to display key metrics 
+
+
+## Built on Metrics Views
+
+All dashboards in Rill are built on [metrics views](/build/metrics-view/metrics-view.md). This ensures that your defined measures and dimensions are standard throughout all of your visualizations. The primary difference between Explore and Canvas dashboards is that **Explore is built on a single metrics view**, while **Canvas can combine multiple metrics views** in one dashboard.
+
+## Explore Dashboards
+
+**[Explore dashboards](/build/dashboards/explore)** are designed for interactive data exploration and analysis. They provide a powerful "slice and dice" experience focused on a single metrics view.
```

Comment:
It'd be helpful to say what visualizations come out-of-the-box in Explore dashboards (e.g. line charts, leaderboards, pivot)